exchange/events.go

```diff
@@ -62,7 +74,11 @@ func (ev *eventTracking) modifyBidVAST(pbsBid *pbsOrtbBid, bidderName openrtb_ex
 		return
 	}
 	vastXML := makeVAST(bid)
-	if newVastXML, ok := events.ModifyVastXmlString(ev.externalURL, vastXML, bid.ID, bidderName.String(), ev.accountID, ev.auctionTimestampMs); ok {
+	bidID := bid.ID
+	if ev.generateBidId {
```

Comment:
Why do we need the check both here and in `modifyBidsForEvents`?

Reply:
how about something like this, to make it less dependent on the unrelated config flag:
```go
func getBidID(pbsBid *pbsOrtbBid) (bidID string) {
        if len(pbsBid.generatedBidId) > 0 {
                bidID = pbsBid.generatedBidId
        else {
                bidID = bid.ID
        }
        return
```

Reply:
Yes. Refactored.